src/core/ext/filters/http/server/http_server_filter.h

```diff
@@ -26,4 +26,9 @@
 /* Processes metadata on the server side for HTTP2 transports */
 extern const grpc_channel_filter grpc_http_server_filter;
 
+namespace grpc_core {
+// Temporary code that allows servers to accept PUT requests.
```

Comment:
Can you make this more discouraging to use? We can put warnings in the comment (something like this will be removed)?

Reply:
Done!